launchdarkly-server-sdk/src/events/event.rs

```diff
@@ -131,21 +131,17 @@ impl Serialize for MigrationOpEvent {
             key: self.key.clone(),
             value: self.evaluation.value,
             default: self.default_stage,
-            // QUESTION: In the ruby implementation, this can be nil. Why not here?
```

Comment:
The ruby one is needlessly checking for nil. 🤦🏼

Reply:
In client-side SDKs the reason is optional, so maybe it is for bootstrapping from ruby? But it would always be populated.